app/src/configurable_auth.h

```diff
@@ -27,6 +28,12 @@ namespace scitt
       auto identity = JwtAuthnPolicy::authenticate(tx, ctx, error_reason);
       if (!identity)
       {
+        CCF_APP_INFO(
+          "ClientRequestId={} Verb={} URL={} ErrorMessage={}",
```

Comment:
We should add `Status=401` here with a comment saying that CCF returns any errors in the auth policy with a 401 status.